src/software/ai/hl/stp/tactic/passer/passer_fsm.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "software/ai/hl/stp/tactic/dribble/dribble_fsm.h"
+#include "software/ai/hl/stp/tactic/kick/kick_fsm.h"
+#include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/ai/intent/move_intent.h"
+#include "software/ai/passing/pass.h"
+
+struct PasserFSM
+{
+    struct ControlParams
+    {
+        // The pass to execute
+        std::optional<Pass> pass = std::nullopt;
+    };
+
+    // this struct defines the only event that the MoveFSM responds to
+    DEFINE_UPDATE_STRUCT_WITH_CONTROL_AND_COMMON_PARAMS
```

Comment:
Typo maybe, in the comment: should be PasserFSM, not MoveFSM?

Reply:
fixed!